pkgs/os-specific/linux/audit/default.nix

```diff
@@ -15,7 +17,7 @@ stdenv.mkDerivation rec {
 
   outputs = [ "bin" "dev" "out" "man" ];
 
-  depsBuildBuild = [ buildPackages.stdenv.cc ];
+  depsBuildBuild = [ buildPackages.stdenv.cc autoconf automake libtool ];
```

Comment:
Will the autofoo dependencies go away after the next release? IMHO add a comment here if so.

Reply:
I suppose they will not go away; in fact I'm not quite sure why it works without them on glibc.

Given that the current `src` is a release tarball, it probably has a pre-generated `configure` script, which should make them unnecessary; but I'm not sure why on musl that one isn't good enough.

Not relying on the pre-generated `configure` script may be generally preferrable by the way, because it allows swapping out the `src` to use e.g. a git repository (which often doesn't have pre-generated `configure`), and is something people often do in overrides to test newer versions.

Reply:
That should not be the case. If it is the case, then upstream might be generating `configure` etc using weird versions of the autotools.

Whether or not every package should depend on autofoo to allow swapping `src` seems like a higher-level question than this package.